R/epiparameter-utils.R

```diff
@@ -484,32 +484,33 @@ create_method_assess <- function(censored = NA,
 #' This check for valid parameters is independent of whether the distribution
 #' is truncated or discretised.
 #'
-#' @inheritParams new_epiparameter
+#' @inheritParams create_prob_distribution
 #'
 #' @return A boolean `logical`.
 #' @export
 #'
 #' @examples
 #' is_epiparameter_params(
-#'   prob_dist = "gamma",
-#'   prob_dist_params = c(shape = 2, scale = 1)
+#'   prob_distribution = "gamma",
+#'   prob_distribution_params = c(shape = 2, scale = 1)
 #' )
-is_epiparameter_params <- function(prob_dist, prob_dist_params) {
-  if (is.na(prob_dist) || anyNA(prob_dist_params)) {
+is_epiparameter_params <- function(prob_distribution,
```

Comment:
The reason I exported it was I thought it might be useful to some users to check if the parameters they want to input are valid for parameterising an `<epiparameter>` as not every possible parameterisation is supported. 

If you think this should be made internal please let me know and I'll unexport it.